e2e/test/scenarios/dashboard-filters/dashboard-filters-auto-apply.cy.spec.js

```diff
@@ -675,7 +675,12 @@ const openSlowDashboard = (params = {}) => {
     "cardQuery",
   );
 
-  visitDashboard("@dashboardId", { params });
+  cy.get("@dashboardId").then(dashboardId => {
+    return cy.visit({
+      url: `/dashboard/${dashboardId}`,
+      qs: params,
+    });
+  });
```

Comment:
For some reason (that I couldn't figure out), every test that uses `cy.clock` in combination with the `visitDashboard` helper fails. It works only when `cy.visit()` is explicitly used.

If anyone has ideas why, I'm all ears.

Reply:
An example of a failed run:
https://github.com/metabase/metabase/actions/runs/7761351435/job/21214199066#step:11:783

And the recording of that failure:
https://app.replay.io/recording/b9a3ad12-f5ff-465b-98be-8a9813df8aa6